DependencyInjection/Configuration.php

```diff
@@ -350,10 +350,86 @@ private function addOrmSection(ArrayNodeDefinition $node)
                     ->append($this->getOrmEntityManagersNode())
                     ->fixXmlConfig('resolve_target_entity', 'resolve_target_entities')
                     ->append($this->getOrmTargetEntityResolverNode())
+                    ->append($this->getOrmSecondLevelCacheNode())
                 ->end()
             ->end()
         ;
     }
+    
+    /**
+     * Return ORM second level node
+     *
+     * @return \Symfony\Component\Config\Definition\Builder\NodeDefinition
+     */
+    private function getOrmSecondLevelCacheNode()
+    {
+        $treeBuilder = new TreeBuilder();
+        $node = $treeBuilder->root('second_level_cache');
+
+        $node
+            ->info('Global second level cache defaults (for all entity managers).')
+            ->children()
+                ->arrayNode('default_cache_factory')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->scalarNode('class')
+                            ->defaultValue('Doctrine\ORM\Cache\DefaultCacheFactory')
+                        ->end()
+                    ->end()
+                ->end()
+                ->arrayNode('default_region')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->scalarNode('class')
```

Comment:
-1 for making all classes configurable in the config. This is the worst extension point ever (even worse than exposing them as DIC parameters, which are already considered a bad practice today, as it bloats the user-facing semantic config). Replacing only the class of a service is very fragile and likely to break when service definition are altered.